### ISSUE TITLE
Theme pages image missing alt attribute

### ISSUE THREAD

**Issue by IO-Fire** [hlassiege/bloggr-demo#5]
Some page images in the mistral theme are missing an alt attribute. This attribute is used by screen readers and search engines to understand the page.

**Bing** - *WARNING : Alt attribute for images is missing*:
> The <img> tag does not have an ALT attribute defined. As a general rule, search engines do not interpret the content of image files. The text provided in the <img alt> attribute enables the site owner to provide relevant information to the search engine and to the end user. Alt text is helpful to end users if they have images disabled or if the image does not properly load. In addition, the Alt text is used by screen readers. Make sure that your Alt text is descriptive and accurately reflects what the image represents and supports the content on the page.

### Examples

- [`components/themes/mistral/ArticleHeader.vue`](https://github.com/bloggrify/bloggrify-mistral/blob/4dde78b69f8c827d9c5c9d0ee65fa0775c2f3b6a/components/themes/mistral/ArticleHeader.vue#L53)
  - [https://mistral.bloggrify.com/about](https://mistral.bloggrify.com/about)
  - [https://mistral.bloggrify.com/features](https://mistral.bloggrify.com/features)

## Potential Fix

Set the `ArticleHeader.vue` image's `alt` attribute to an appropriate value.
